ci/Dockerfile-nighthawk-output-transform

```diff
@@ -0,0 +1,5 @@
+FROM frolvlad/alpine-glibc
```

Comment:
Do we really want a separate image for each tool? Can't we just put them in one? I imagine the size difference isn't significant.

Reply:
+1, pushed a change that uses a single image in f6478cea7880a077e441d86b017c053d6efcb7f4
The binary sizes look rather large though, looking into that.
@howardjohn binary size aside, any objections to this moving all the tools into a single image?

Reply:
Nope, seems like a good idea to me

Reply:
Sounds good to me

Reply:
great, added 1a30e966748f1da530f7b17880b52f1e5c17b7a6 to strip the binaries